### ISSUE TITLE
Allow for integration of fieldset duplication

### ISSUE THREAD

**Issue by kgollmer** [markocupic/calendar-event-booking-bundle#53]
Hey there,
while I do love this slick and lightweight add-on, I really do miss some features, most pressingly:
Please allow for integration of https://github.com/inspiredminds/contao-fieldset-duplication – I do think this will help other people as well, since it is a really nifty function, especially if you want to allow for one person to register multiple people.

**Comment by markocupic** [comment]
Hi @kgollmer 
Please feel free and add your own pull request to the application.

**Comment by kgollmer** [comment]
Ouf, ok, I'll try to figure out how to do that 😅

**Comment by markocupic** [comment]
Of course, it is also possible to sponsor the further development of the extension.

**Comment by tognit** [comment]
And how could the sponsorship look like? ;-)
